src/repository/SchedulesRepository.ts

```diff
@@ -72,7 +72,7 @@ export class SchedulesRepository extends Repository<ScheduleEntity> {
 
   async createIndex(): Promise<void> {
     // this unique index combined with the always same `name` value of every schedule, ensures
```

Comment:
I guess the first line of this comment is a bit misleading since the `name` value is not always the same for every schedule now.